File: beginner/javascript/1158.js
var input = require('fs').readFileSync('stdin', 'utf8');
var linhas = input.split('\n');

var i, valores, n, cont = 0, soma;

n = parseInt(linhas.shift());

for (i = 0; i < n; i++) {
    soma = 0;
    valores = linhas.shift().split(" ");
    var x = parseInt(valores[0]);
    var y = parseInt(valores[1]);

    while (cont < y) {
        if (x % 2 !== 0) {
            soma += x;
            cont++;
        }
        x++;
    }

    console.log(soma);
    cont = 0; 
}

File: beginner/python/1158.py
cont = 0

n = int(input())

for i in range(0,n):
    soma = 0
    x, y = list(map(int, input().split()))

    while cont < y:
        if (x % 2 != 0):
            soma += x
            cont+=1
        
        x+=1

    print(soma)
    cont = 0 